backend/src/config/app-config.ts

```diff
@@ -0,0 +1,18 @@
+import {AppProperties} from './app-properties.model';
+export class AppConfig {
+
+  private _appConfig: AppProperties;
+
+  // configName is the name of the properties file.
+  // There's an untracked folder properties at the same level as the src directory with the properties.
+  public setAppConfig(configName: string) {
+    this._appConfig = require(`../../properties/${configName}.properties.json`);
+  }
+
+  public get appConfig(): AppProperties {
+    return this._appConfig;
+  }
+
+}
+
+export const appConfig = new AppConfig();
```

Comment:
I would rather see this as an Nest Injectable however I don't have an idea how to organise the config factory which should use process args - hardcoding is not a best idea, propably creating Injectable on bootstrap might work, don't know 😕